code/controllers/subsystems/statpanel.dm

```diff
@@ -0,0 +1,117 @@
+SUBSYSTEM_DEF(statpanels)
+	name = "Stat Panels"
+	wait = 4
+	init_order = INIT_ORDER_STATPANELS
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+	var/list/currentrun = list()
+	var/encoded_global_data
+	var/mc_data_encoded
+	var/list/cached_images = list()
+
+/datum/controller/subsystem/statpanels/fire(resumed = FALSE)
+	if (!resumed)
+		var/datum/map_config/cached = SSmapping.next_map_config
+		var/round_time = world.time - SSticker.round_start_time
+		var/list/global_data = list(
+			"Map: [SSmapping.config?.map_name || "Loading..."]",
+			cached ? "Next Map: [cached.map_name]" : null,
+			"Round ID: [GLOB.round_id ? GLOB.round_id : "NULL"]",
+			"Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss")]",
+			"Round Time: [round_time > MIDNIGHT_ROLLOVER ? "[round(round_time/MIDNIGHT_ROLLOVER)]:[worldtime2stationtime()]" : worldtime2stationtime()]",
+			"Station Time: [stationtime2text()]",
+			"Time Dilation: [round(SStime_track.time_dilation_current,1)]% AVG:([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)"
+		)
+
+		if(SSshuttle.emergency)
+			var/ETA = SSshuttle.emergency.getModeStr()
```

Comment:
Most likely going to be need to be changed to
```c
var/ETA = emergency_shuttle_controller.emergency_shuttle.get_status_panel_eta()
```

See: https://github.com/VOREStation/VOREStation/blob/5195f715452d7a92f2b911f8217343a856f6ed13/code/controllers/emergency_shuttle_controller.dm#L227